src/dmd/declaration.d

```diff
@@ -475,7 +475,7 @@ extern (C++) abstract class Declaration : Dsymbol
         return (storage_class & STC.parameter) != 0;
     }
 
-    override final bool isDeprecated() const pure nothrow @nogc @safe
+    override final bool isDeprecated() pure nothrow @nogc @safe
```

Comment:
?

Reply:
Yep, the frontend apparently doesn't care about this invalid override: https://github.com/dlang/dmd/blob/master/src/dmd/dsymbol.d#L782

Reply:
Could the base method have const added instead?

Reply:
> invalid override

It's a feature, not a bug. It's the miracle of covariance :-)